### DISCUSSION TITLE
Upload Recordings to AWS S3

### DISCUSSION THREAD

**Discussion by lahirudx** [Kurento/kurento#22] [Community]
How to configure RecorderEndpoint to save recordings to AWS S3 storage?

**Comment by j1elo** [comment]
RecorderEndpoint has a file writer and an HTTP stream writer, but not an S3 object writer; Kurento had one at some point, but it was proprietary and their owners removed access to it.

**Reply by lahirudx** [reply]
Thanks for letting me know. I have created a separate node app to do the uploading task.
